tests/infrastructure/instance_types/test_vm_preference.py

```diff
@@ -68,9 +68,7 @@ def fedora_data_volume_template(dv_template_api, golden_images_fedora_data_sourc
     fedora_dv_template = data_volume_template_with_source_ref_dict(data_source=golden_images_fedora_data_source)
     if dv_template_api == "pvc":
         return pvc_api_adjustments(dv_template=fedora_dv_template)
-    else:
-        del fedora_dv_template["spec"]["storage"]["storageClassName"]
```

Comment:
If the source is a PVC, we still need to delete the `storageClassName`, right?

Reply:
These are different things, let me try to explain
pvc api - means a path in a `DV.spec.[pvc/storage]`
PVC as a source means that the image is stored in the PVC object, instead of a VolumeSnapshot object

Reply:
The source is always a DataSource (which in 4.19 is a VolumeSnapshot)
Those changes apply to the API we use, and also the naming of the function (which no longer only apply to PVC)

I don't understand your question, please explain what do you think should be changed in the PR?

Reply:
DataSource may point either to a `PVC`, or to a `VolumeSnapshot`. 
Which one - it depends on your default storage class and what is has in the storageProfile.
For example, if `hostpath-csi-basic` is your default storage, the DataSource will point to a PVC
```
$ oc get storageprofile hostpath-csi-basic -oyaml | grep dataImportCronSourceFormat
  dataImportCronSourceFormat: pvc

```